third_party/move/move-vm/runtime/src/storage/loader.rs

```diff
@@ -1,10 +1,8 @@
 // Copyright (c) The Move Contributors
 // SPDX-License-Identifier: Apache-2.0
 
-use crate::{config::VMConfig, loader::LoadedFunctionOwner, CodeStorage, LoadedFunction};
-use move_binary_format::errors::{Location, PartialVMResult, VMResult};
-use move_core_types::language_storage::TypeTag;
-use move_vm_types::loaded_data::runtime_types::{Type, TypeBuilder};
+use crate::config::VMConfig;
+use move_vm_types::loaded_data::runtime_types::TypeBuilder;
 
 /// V2 implementation of loader, which is stateless - i.e., it does not contain module or script
 /// cache. Instead, module and script storages are passed to all APIs by reference.
```

Comment:
haha, do we even need this LoaderV2 wrapper at all?

Reply:
We don't, I am removing it further down the stack of PRs👍